docs/src/notebooks/2_finding_balance.jl

```diff
@@ -70,7 +70,7 @@ dict_soln = flux_balance_analysis_dict(
         change_objective("R_BIOMASS_Ecoli_core_w_GAM"),
 
         ## this fixes a specific rate of the glucose exchange
-        change_constraint("R_EX_glc__D_e", -12, -12),
+        change_constraint("R_EX_glc__D_e"; lb = -12, ub = -12),
```

Comment:
isn't this completely complementary to the systematic change_bounds functions that we have now?

Reply:
ah these are modifications not variants. Maybe it would be nice to have the same labeling there, in `change_bounds` we have `lower` and `upper` which I find a bit more readable.

Reply:
hmm the problem is that we are pretty inconsistent with `lb` vs `lower` vs `xl`. I really think we need to change them all to the same thing. I think `lb` is better since it is a `lower_bound`... Maybe we should do this sweeping change asap... it is a pretty big breaking change but the extra consistency would be nice

Reply:
`xl` is the matrix naming, `lower_bound` is the normal naming, `lower` is a natural shortcut if you already have `bound` in the function name (can apply for `constraint` too I'd say)